server/config/passport.js

```diff
@@ -1,13 +1,19 @@
-const DiscordStrategy = require("passport-discord").Strategy;
+const DiscordStrategy = require("passport-discord");
```

Comment:
This seems like a breaking change?  Is the default export of `passport-discord` also a named export `Strategy`?

Reply:
I messed up, but it's still working for some reason. Probably some extension interferes with imports. Will fix it